ext/Pod-Html/lib/Pod/Html/Util.pm

```diff
@@ -30,75 +30,52 @@ Pod::Html::Util - helper functions for Pod-Html
-                       'htmlroot=s' => \$opt_htmlroot,
-                       'index!'     => \$opt_index,
-                       'infile=s'   => \$opt_infile,
-                       'outfile=s'  => \$opt_outfile,
-                       'poderrors!' => \$opt_poderrors,
-                       'podpath=s'  => \$opt_podpath,
-                       'podroot=s'  => \$opt_podroot,
-                       'quiet!'     => \$opt_quiet,
-                       'recurse!'   => \$opt_recurse,
-                       'title=s'    => \$opt_title,
-                       'verbose!'   => \$opt_verbose,
+    my $result = GetOptions(\%opts,
+        'backlink!',
+        'cachedir=s',
+        'css=s',
+        'flush',
+        'help',
+        'header!',
+        'htmldir=s',
+        'htmlroot=s',
+        'index!',
+        'infile=s',
+        'outfile=s',
+        'poderrors!',
+        'podpath=s',
+        'podroot=s',
+        'quiet!',
+        'recurse!',
+        'title=s',
+        'verbose!',
     );
     usage("-", "invalid parameters") if not $result;
-
-    usage("-") if defined $opt_help;    # see if the user asked for help
-    $opt_help = "";                     # just to make -w shut-up.
-
-    @{$globals->{Podpath}}  = split(":", $opt_podpath) if defined $opt_podpath;
-
-    $globals->{Backlink}  =          $opt_backlink   if defined $opt_backlink;
-    $globals->{Cachedir}  =  unixify($opt_cachedir)  if defined $opt_cachedir;
-    $globals->{Css}       =          $opt_css        if defined $opt_css;
-    $globals->{Header}    =          $opt_header     if defined $opt_header;
-    $globals->{Htmldir}   =  unixify($opt_htmldir)   if defined $opt_htmldir;
-    $globals->{Htmlroot}  =  unixify($opt_htmlroot)  if defined $opt_htmlroot;
-    $globals->{Doindex}   =          $opt_index      if defined $opt_index;
-    $globals->{Podfile}   =  unixify($opt_infile)    if defined $opt_infile;
-    $globals->{Htmlfile}  =  unixify($opt_outfile)   if defined $opt_outfile;
-    $globals->{Poderrors} =          $opt_poderrors  if defined $opt_poderrors;
-    $globals->{Podroot}   =  unixify($opt_podroot)   if defined $opt_podroot;
-    $globals->{Quiet}     =          $opt_quiet      if defined $opt_quiet;
-    $globals->{Recurse}   =          $opt_recurse    if defined $opt_recurse;
-    $globals->{Title}     =          $opt_title      if defined $opt_title;
-    $globals->{Verbose}   =          $opt_verbose    if defined $opt_verbose;
-
-    warn "Flushing directory caches\n"
-        if $opt_verbose && defined $opt_flush;
-    $globals->{Dircache} = "$globals->{Cachedir}/pod2htmd.tmp";
-    if (defined $opt_flush) {
-        1 while unlink($globals->{Dircache});
-    }
-    return $globals;
+    usage("-") if defined $opts{help};  # see if the user asked for help
+    $opts{help} = "";                   # just to make -w shut-up.
+    return \%opts;
 }
 
 =head2 C<usage()>
 
-TK
+Provide usage message on STDERR.
```

Comment:
```suggestion
Print usage message to STDERR.
```

Reply:
Between the time of your initial review and now I committed other changes to the documentation of the helper subroutines in `lib/Pod/Html.pm`.  So you should probably work from those.